<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bookman Old Style</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagenes/avatar-1299805_1280.png</normaloff>imagenes/avatar-1299805_1280.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(128, 128, 128)</string>
  </property>
  <widget class="QPushButton" name="bttnInicio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tempus Sans ITC</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 114, 167);
border-radius: 5px
</string>
   </property>
   <property name="text">
    <string>Iniciar sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 71, 104);</string>
   </property>
   <property name="text">
    <string>USUARIO:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 71, 104);</string>
   </property>
   <property name="text">
    <string>CONTRASEÑA:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:10px;
color: rgb(0, 0, 0);
border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="placeholderText">
    <string>NOMBRE DE USUARIO...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>Usuario</string>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:10px;

border:none
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>CONTRASEÑA...</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btncerrar1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
font: 81 9pt &quot;Rockwell Extra Bold&quot;;
border:1px solid black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
